server/src/routes/api/v1/planetReviewsRouter.js

```diff
@@ -0,0 +1,38 @@
+import express from "express"
+import objection from "objection"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+const { ValidationError } = objection
+import { Review } from "../../../models/index.js"
+
+const planetReviewsRouter = new express.Router({ mergeParams: true })
+
+planetReviewsRouter.post("/", async (req, res) => {
+  const { body } = req
+  const formInput = cleanUserInput(body)
+  const { reviewText } = formInput
+  const { planetId}  = req.params
+  console.log(req.params)
+
+  try {
+    const review = await Review.query().insertAndFetch({ reviewText, planetId })
```

Comment:
need to also ensure the userid is saved with the review, and you need to ensure only signed in users can make a review